<?xml version="1.0"?>
<launch>
  <arg name="footprint_file" default="ERROR: did not specify argument --footprint_file--" />
  <arg name="scan_topics" default="ERROR: did not specify argument --scan_topics--" />
  <arg name="target_frame" default="/base_link" />

  <node pkg="mcr_navigation_tools" type="laser_distances_node" name="laser_distances" ns="mcr_navigation" output="screen">
    <rosparam file="$(arg footprint_file)" command="load"/>
    <rosparam file="$(find mcr_collision_velocity_filter)/ros/config/collision_velocity_filter.yaml" command="load"/>

    <rosparam param="scan_topics" subst_value="True">$(arg scan_topics)</rosparam>
    <rosparam param="target_frame" subst_value="True">$(arg target_frame)</rosparam>
    <param name="loop_rate" type="double" value="10.0" />

  </node>

</launch>
